packages/gatsby-theme-newrelic/src/components/SimpleFeedback.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql, useStaticQuery } from 'gatsby';
+import { css } from '@emotion/core';
+
+import Button from './Button';
+import Icon from './Icon';
+import PageTools from './PageTools';
+
+const SimpleFeedback = ({ title, slug, labels }) => {
+  const { site } = useStaticQuery(graphql`
+    query FeedbackQuery {
+      site {
+        siteMetadata {
+          siteUrl
+          repository
+        }
+      }
+    }
+  `);
+
+  const { repository, siteUrl } = site.siteMetadata;
+  const issueUrl = `${repository}/issues/new`;
+
+  const issueTitle = title
+    ? `Feedback:+${title.replace(' ', '+')}`
+    : 'Website Feedback';
+
+  const body =
+    title && slug ? `&body=Page:%20[${title}](${siteUrl}${slug})` : '';
+
+  const positiveFeedback = `${issueUrl}?labels=${labels.join(
+    ','
+  )},feedback-positive&title=${issueTitle}${body}`;
```

Comment:
In the event that `labels` is empty, this might add an extra comma that you don't intend. You might consider appending the default label to the `labels` array and then `.join`

```js
`labels=${[...labels, 'feedback-negative'].join(',')}
```

Reply:
Good catch!